scripts/install_archiver.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# This script goal is to deploy the archiver service container in the host
+
+readonly USAGE="usage: archiver.sh <conf_files_path>"
+readonly ARCHIVER_IMAGE=ufcgsaps/archiver
+readonly CONTAINER_NAME=archiver
+readonly NETWORK_NAME=saps_net
+readonly TEMP_STORAGE_DIR=/nfs
+
+readonly CONF_FILE=archiver.conf
+readonly LOG4J_FILE=log4j.properties
+readonly EXECUTION_TAGS_FILE=execution_script_tags.json
+
+IMAGE_TAG=latest
+
+### Checks for all configuration files
+check_conf_dir() {
+  local CONF_FILES_PATH=${1}
+  if [ ! -d ${CONF_FILES_PATH} ] 
+  then
+      echo "Directory ${CONF_FILES_PATH} DOES NOT exists." 
+      exit 1
+  fi
+  for FILE in ${CONF_FILE} ${LOG4J_FILE} ${EXECUTION_TAGS_FILE}
+  do
+    if [[ ! -f "${CONF_FILES_PATH}/${FILE}" ]]; then
+      echo "${FILE} was not found in the ${CONF_FILES_PATH} dir!"
+      exit 1
+    fi
+  done
+}
+
+get_absolute_path() {
+  local PATH=${1}
+  echo $(cd ${PATH}; pwd)
+}
+
+# Checks if the container already exists
+container_exists() {
+  CHECK=$(docker ps -aq -f name="${CONTAINER_NAME}")
+  if [ ! -z "${CHECK}" ]; then
+    echo "The container ${CONTAINER_NAME} already exists"
```

Comment:
but you'll remove it, right? so, tell it to the user: "..already exists, removing it...

Reply:
The remove_service func log already shows that it will be removed.